src/styles/product-card.scss

```diff
@@ -0,0 +1,104 @@
+.product-card {
+  width: 200px;
+  height: 406px;
```

Comment:
You shouldn't hardcode container's height. Let the content dictate it
![image](https://github.com/mate-academy/layout_product-cards/assets/96409563/959fb025-5ed4-45dd-b46c-960bbd7f1724)